packages/libs/client/etc/client.api.md

```diff
@@ -432,6 +443,11 @@ export class Literal {
 // @public
 export const literal: (value: string) => Literal;
 
+// @alpha (undocumented)
+export type Milliseconds = number & {
```

Comment:
Is this something that we want and use more often for primitives that are used in a specific context?

I'd like to talk about this a little bit, as there are various ways to do this. Maybe we want to use symbols?

Does a standard exist in the community that we can follow?

Reply:
Yes using `brand` is a kind of standard in TS community. I made that optional here, so it works mostly like a hint to user.
Since number is very generic having this alias will make the type more clear.